src/сomponents/App/App.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import { render } from "react-dom";
+import FirstType from "../Experimental/FirstType/FirstType";
+import { SecondType } from "../Experimental/SecondType";
+import ThirdType from "../Experimental/ThirdType";
+import FourthType from "../Experimental/FourthType";
+
+export default class App extends Component {
+  constructor() {
```

Comment:
Why do you need a constructor here?

Reply:
I don't need it, I think I saw some tutorial, where adding constructor was considered as "best practice". I guess I was wrong